tests/test_device.py

```diff
@@ -84,3 +85,25 @@ def test_U8() -> None:
 #     assert not ser.is_open
 #
 #     # assert data[0] == '\t'
+
+
+def test_device_events(device: Device) -> None:
+
+    event_q = device._ser.event_q
+
+    while True:
```

Comment:
You have a nice `get_events()` function already written. Can we use that here in this example instead of accessing the underlying object?